src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/ReadyToRunCodegenCompilation.cs

```diff
@@ -264,8 +276,13 @@ protected override void ComputeDependencyNodeDependencies(List<DependencyNodeCor
         {
             using (PerfEventSource.StartStopEvents.JitEvents())
             {
-                ConditionalWeakTable<Thread, CorInfoImpl> cwt = new ConditionalWeakTable<Thread, CorInfoImpl>();
-                foreach (DependencyNodeCore<NodeFactory> dependency in obj)
+                ParallelOptions options = new ParallelOptions
+                {
+                    MaxDegreeOfParallelism = _parallelism
+                };
+                Console.WriteLine("Parallelism");
+                Console.WriteLine(_parallelism);
```

Comment:
I'm guessing these WriteLines were for debugging purposes? :)

Reply:
Yep